simplic-user/Simplic.User.UI/Views/OrganizationDetailsView.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<simplic:DefaultRibbonWindow x:Class="Simplic.User.UI.OrganizationDetailsView"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:simplic="http://schemas.simplic-systems.com/2016/xaml/presentation"
+        Title="{Binding Path=Organization.Name}" Height="120" Width="400"
+        WindowStartupLocation="CenterOwner" LayoutContext="Win_OrganizationDetails">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <DockPanel Margin="7,10,7,7" LastChildFill="True">
+            <Label DockPanel.Dock="Left" Content="Organization Name:" />
+            <TextBox Margin="5,0,0,0" Text="{Binding Path=NewOrganizationName, Mode=TwoWay}" Height="27" VerticalAlignment="Top" VerticalContentAlignment="Center" />
```

Comment:
We always use simplic:TextBox, because it contains some custom behaviour.

Reply:
Done